TestProgram/Benchmarks/speed_test_reading_gpu.c

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+*
+* NOCOMPILE= !WITH_GPU
```

Comment:
shouldn't this test work also on CPU?

Reply:
It does but it is an unfinished draft for a GPU memory access speed test, that I intent to change to something better. If its a problem that this is there, I will remove this file for now. If we want a linear algebra cpu/gpu comparison speed test, this should be maybe designed differently.

Reply:
ok. no problem.